src/ripple/protocol/impl/Feature.cpp

```diff
@@ -186,7 +187,8 @@ uint256 const
     featureHardenedValidations      = *getRegisteredFeature("HardenedValidations"),
     fixAmendmentMajorityCalc        = *getRegisteredFeature("fixAmendmentMajorityCalc"),
     featureNegativeUNL              = *getRegisteredFeature("NegativeUNL"),
-    featureTicketBatch              = *getRegisteredFeature("TicketBatch");
+    featureTicketBatch              = *getRegisteredFeature("TicketBatch"),
+    fixOfferSequence                = *getRegisteredFeature("fixOfferSequence");
```

Comment:
Outside the scope of this PR but I wanted to note the concern. These string literals are used in three different places, which leaves room for misspells or incorrect camel-case. It may be a good idea to define and reference the text in a single place.